src/components/BookHook.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const BookHook = () => {
```

Comment:
it seems to me like you repeated the same div over and over again

Reply:
![image](https://github.com/user-attachments/assets/34adf42c-c199-4e38-98bc-9d99ea892075)

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from](Array.from)

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map](map)

Reply:
for now you can do this 

![image](https://github.com/user-attachments/assets/f1484e2e-12de-4d7e-baca-2ef0fb1850bd)